.config/webpack/base.js

```diff
@@ -19,6 +19,7 @@ module.exports = (preprocessor = {}, mode = 'development') => {
             path: path.resolve(__dirname, `./../../${dashLibraryName}`),
             filename: '[name].js',
             library: dashLibraryName,
+            publicPath: '/_dash-component-suites/dash_table/',
```

Comment:
Dynamically loaded files expect to be in the same path as the requester -- this is not true in Dash. I hate breaking this assumption -- creates a deep (and mostly unknown) dependency between Dash BE logic and the component build.

Reply:
(for completeness: lazy loading was removed in https://github.com/plotly/dash-table/pull/338/commits/cd277ed66dfd96ac1b774aaa825222b2126cf6ee pending discussion in https://github.com/plotly/dash-core/issues/39)